src/pages/index.js

```diff
@@ -1,15 +1,17 @@
 import Section from "../components/Section.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
Please, delete `this.disableButton();` from line 79 in `FormValidator`

You need to **disable** the button only in the `then` block after a **successful response** to be able to click it again if there is a server error.